packages/amplication-client/src/Purchase/FAQ.tsx

```diff
@@ -19,115 +19,118 @@ export const FAQ = () => {
+                  transition from monolithic architectures to microservices,
+                  allowing for the creation of scalable and manageable backends.
+                </li>
+                <li>
+                  <strong>Customizable Code Generation</strong>: Ensures
+                  flexibility in development, allowing for specific
+                  customizations without being bound to rigid system
+                  constraints.
+                </li>
+              </ul>
+            </div>
+          }
         />
         <Question
-          question="Can I pay for my subscription with an invoice?"
+          question="Can you assist with the actual deployment (terraform-like) in a cloud provider?"
           answer={
             <div>
-              Companies can use invoices to purchase Enterprise subscriptions.
-              Please{" "}
-              <MailTo
-                email="mailto:sales@amplication.com"
-                subject="Amplication Enterprise Plan Inquiry"
-                body="contact sales"
-              />{" "}
-              for more information and to purchase.
+              Amplication allows you to deploy to AWS's Elastic Container
+              Service (ECS). We provide in-depth documentation that will walk
+              you through using various Amplication plugins to provision the
+              required infrastructure and set up a process for continuous
+              integration and deployment.
+              <br />
+              <br />
+              The steps cover manual configurations for permissions and
+              authentication, plugin settings for AWS services, code generation,
+              provisioning with Terraform, and deployment using GitHub Actions.
             </div>
           }
         />
         <Question
-          question="We're an open-source project, is there special pricing available?"
+          question="I have an existing DB with a lot of data, entities, tables, and relationships. Can I use your backend development platform, while still keeping the data and the knowledge?"
+          answer="Amplication's DB Schema Import feature provides a clear pathway for integrating an existing database with the Amplication platform. The import feature is available on all Amplication plans, including Free accounts."
+        />
+        <Question
+          question="Will your generated code override my code changes?"
           answer={
             <div>
-              Yes, as part of our commitment to support open-source projects, we
-              offer the pro plan for free. Reach out{" "}
-              <MailTo
-                email="mailto:sales@amplication.com"
-                subject="Pro plan for Open Source project"
-                body="here"
-              />{" "}
-              and we will set you up.
+              Amplication's Smart Git Sync feature ensures that changes made in
```

Comment:
I think the answer should be much more clear and short- "Yes, Amplication's Smart Git Sync feature ensures...", we always prefer your custom code over the generated code, and know how to combine the two automatically.
Something like that. A link to the documentation should be considered.